addons/website/static/tests/tours/page_manager.js

```diff
@@ -47,7 +47,13 @@ const checkWebsiteFilter = [{
 	content: "Select Test Website",
 	trigger: ".o_dropdown_container.o_website_menu > .dropdown-item:contains('Test Website')",
     run: "click",
-}, {
+    },
+    {
+        content: "Check if active website facet shows 'Test Website'",
+        trigger:
+            ".o_searchview_input_container .o_searchview_facet.o_facet_active_website small.o_facet_value:contains('Test Website')",
```

Comment:
```suggestion
            ".o_searchview_facet small.o_facet_value:contains('Test Website')",
```

let's simplfy the selector for the facet value. ( apply the same other place )